modal/runner.py

```diff
@@ -105,7 +106,7 @@ async def _create_all_objects(
     new_app_state: int,
     environment_name: str,
     output_mgr: Optional[OutputManager] = None,
-):  # api_pb2.AppState.V
+) -> None:  # api_pb2.AppState.V
```

Comment:
i guess the comment is incorrect here

Reply:
https://github.com/modal-labs/modal-client/pull/1875/commits/5a78205832713fadc0dc34d97f3cac9744e5f155